test.cr

```diff
@@ -0,0 +1,8 @@
+name = if true
+         ""
+       else
+         nil
+       end
+puts typeof(name) # => (String | Nil)
+name || abort "that's nil!"
+puts typeof(name) # => String
```

Comment:
?

Reply:
It was never here *waves hand*
I had copy-pasted that snippet into a file to run it, and it must've accidentally gotten committed when I was syncing my work from one computer to another

Reply:
I do this sometimes too